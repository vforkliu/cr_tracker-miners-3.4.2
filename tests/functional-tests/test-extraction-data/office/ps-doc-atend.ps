%!PS
%%Copyright: copyleft
%%Title: Hello World!
%%Creator: vi
%%Pages: (atend)
%%EndComments
/Courier
20 selectfont
72 500 moveto
(Hello world!) show
showpage
%%Trailer
%%Pages: 1
