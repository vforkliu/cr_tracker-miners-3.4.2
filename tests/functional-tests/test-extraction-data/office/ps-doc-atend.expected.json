{
    "test": {
        "Filename": "ps-doc-atend.ps",
        "Comment": "PS document from the office tools"
    },
    "metadata": {
        "nfo:pageCount": 1,
        "nie:copyright": "copyleft",
        "nco:creator": {
            "nco:fullname": "vi",
            "@type": "nco:Contact"
        },
        "@type": "nfo:PaginatedTextDocument"
    }
}